src/components/LandingHero.vue

```diff
@@ -0,0 +1,220 @@
+<template>
```

Comment:
I believe this component still has too many lines. Should we extract something more or leave it as it is?